### ISSUE TITLE
Bug in `sumt` for species tree analysis

### ISSUE THREAD

**Issue by nylander** [NBISweden/MrBayes#116]
This error occurs when `sumt` is summarizing the species tree:

```
   Summarizing trees in files "finch.nex.tree31.run1.t" and "finch.nex.tree31.run2.t"
   Using relative burnin ('relburnin=yes'), discarding the first 25 % of sampled trees
   Writing statistics to files finch.nex.<parts|tstat|vstat|trprobs|con>
   Examining first file for tree 30 ...
   Found one tree block in file "finch.nex.tree31.run1.t" with 21 trees in last block
   Expecting the same number of trees in the last tree block of all files

   Tree reading status for tree 31:

   0      10      20      30      40      50      60      70      80      90     100
   v-------v-------v-------v-------v-------v-------v-------v-------v-------v-------v
   ********   Taxon 1 should not be in sampled tree
   Error in command "Tree"
   Error in command "Tree"

```

In this tree (nr 31, which is the species tree) the taxon labels are, and should be, different
from all other (gene) trees. It appears that MrBayes think they should be the same.

Offending code (in `sumpt.c`):

```C
/* check whether we have some taxa that should not be in the tree */
for (i=0; i<t->nNodes; i++)
    {
    p = t->allDownPass[i];
    if (p->left == NULL && taxaInfo[p->index].isDeleted == NO && sumtParams.absentTaxa[p->index] == YES)
        {
        MrBayesPrint ("%s   Taxon %d should not be in sampled tree\n", spacer, p->index + 1);
        return (ERROR);
        }
    }
```

#### Commands to repeat

```
$ wget https://raw.githubusercontent.com/NBISweden/MrBayes/develop/examples/finch.nex
$ mb -i finch.nex
MrBayes > mcmc ngen=10000 nchains=1
MrBayes > sumt
```

#### MrBayes version

3.2.7a (branch develop, 2019-July-24)

```
$ mb -v
MrBayes, Bayesian Analysis of Phylogeny

Version:   3.2.7a
Features:  SSE AVX Beagle readline
Host type: x86_64-unknown-linux-gnu (CPU: x86_64)
Compiler:  gnu 7.4.0
```

**Comment by nylander** [comment]
A work-around to complete the analysis is to run `sumt` separately on the generated tree files, using, for example, a data file with the same taxon labels as in the species tree:

```
$ wget https://raw.githubusercontent.com/NBISweden/MrBayes/develop/examples/finch.nex

$ mb -i finch.nex

MrBayes > mcmc ngen=10000 nchains=1
MrBayes > sumt
MrBayes > quit

$ cat << ADHOC > finch.nex.tree31
#NEXUS
Begin data;
Dimensions ntax=4 nchar=1;
Format datatype=DNA gap=- missing=?;
Matrix
SpQ A
SpW A
SpB A
SpO A
;
end;
begin mrbayes;
set autoclose=yes nowarnings=yes;
sumt;
end;
ADHOC

$ mb -i finch.nex.tree31
```

**Comment by andkaha** [comment]
This is supposedly fixed with PR #152. I'm testing it now before merging it.

**Comment by nylander** [comment]
Also changed the manual according to the fix of #116 and pull request #152 .

**Comment by kusalananda** [comment]
> Also changed the manual according to the fix of #116 and pull request #159.

#159 is not a PR number.
